msteams-platform/apps-in-teams-meetings/teams-together-mode.md

```diff
@@ -0,0 +1,239 @@
+
+> [!VIDEO https://www.youtube-nocookie.com/embed/MGsNmYKgeTA]
+
+You can create scene only apps or extend your current app to have scenes for Together Mode. A scene is a virtual view of the auditorium or meeting venue where the attendees are placed and includes the seats for attendees.
+
+>[!NOTE]
+> Scene only apps are recommended as the acquisition experience for such apps is more seamless.
+
+The following image gives an overview of adding a scene to an app:
+
+![Add scene to app](../assets/images/apps-in-meetings/scene-added-to-app.png)
+
+>[!NOTE]
+> Multiple scenes in a single app package appear as a flat list of scenes to users.
+
+There are prerequisites that you must complete before using the Together Mode.
+
+## Prerequisites
+
+You must have a basic understanding of the following to use Together Mode:
+
+* Scene
+* Scene Design Studio
+* Create apps in App Studio
+* Sideload or upload apps in Teams
+
+Now you can create or build a scene in the Scene Design Studio using Scene editor, where you can add meeting participants, add background images, and align images to the scene.
+
+## Build a scene using the Scene Design Studio
+
+Microsoft has a Scene Design Studio that allows you to build the scenes. It is available on the [Teams Dev Center](https://dev.teams.microsoft.com/home).
+
+Together Mode allows you to apply different scenes while conducting meetings in a virtual environment.
+
+A scene is an artifact that contains the following:
+
+* Seats of meeting participants if their videos are turned on.
+* Each seat is placed on an XY canvas.
+* A collection of images that are camouflaged as one image.
+* Each seat and image has width and height.
+* Optionally, you can reserve a spot for the meeting organizer and a set of reserved spots for participants.
+
+![Scene Design Studio](../assets/images/apps-in-meetings/scene-design-studio.png)
+
+In the Scene Design Studio, each seat represents an avatar for design purposes. Each seat and image has a placement in the third dimension, that is the Z-axis to give a sense of depth in scenes that require them.
+
+![Build a scene](../assets/images/apps-in-meetings/build-a-scene.png)
+
+**To build a scene using the Scene Design Studio**
+
+1. Go to [Teams Dev Center](https://dev.teams.microsoft.com/home).
+
+2. Select **Tools** from the left-hand section and then select **Scene editor** under **Tools** to open the Scene Design Studio.
+
+3. From the **Scenes Editor** page, select **Create a new scene**.
+
+4. Select **Participants** under **Layers** in the upper-right corner.
+
+5. Select the number of participants for the scene from the **Number of participants** box, and select **Add**. After the scene is shipped, the avatar placements are replaced with actual participant's video streams.
+
+6. Drag and drop the image into the environment as displayed in the following image:
```

Comment:
This step should come before step 4

Reply:
I @ mentioned Deborah in a thread. Can we get those assets attached here so that anybody trying this out can simply download the asset and can try it out themselves as they read through the doc.

Reply:
Please suggest how to add the assets is it by creating a folder in apps-in-teams-meetings and attaching Sample scene1 and scene2 zip files.

Reply:
Added in step 5 as note please check

Reply:
Can we rename the assets to Sample Scene.zip and Sample App.zip.